README.md

```diff
@@ -7,6 +7,81 @@ CeresMeta is the meta service for managing the CeresDB cluster.
 ## Status
 The project is in a very early stage.
 
+## Quick Start
+### Build ceresmeta binary
+```
+go build -o ceresmeta cmd/meta/main.go
+```
+
+### Start server(single node)
```

Comment:
Add one line description below:
```plaintext
Although CeresMeta is designed to deployed as a cluster with three or more instances, it can also be started standalone.
```